service/Auth/index.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+const constants = require("../../model/constants");
+const callbacks = require("../../utils/callbacks");
+const AuthConf = require("./AuthConf");
+const passport = require("passport");
+const crypto = require("../../utils/crypto");
+const session = require("../../service/Users/sessions");
+const token = require("../../utils/token");
+class Auth extends AuthConf {
```

Comment:
Good ! but thinking we can do with some other approach , without extending. could. you please highlight your thought why you want to use inheritance?

Reply:
we used inheritance for the case where we can do some post operation on the imported configuration which is not specifically constant in most of the cases